### DISCUSSION TITLE
How to add a `y=x` line?

### DISCUSSION THREAD

**Discussion by jaanaltosaar** [uwdata/mosaic#152] [General]
At @onefact we are building on Mosaic for dashboards of hospital prices, like this one: https://beta.payless.health/examples/mount-sinai.html 

YAML that generates this: https://github.com/onefact/beta.payless.health/blob/main/docs/public/specs/yaml/mount-sinai.yaml

I've read through the docs and can't figure out how to add a line for `y = x` (on the diagonal) to illustrate what an efficient market should look like. 

I see the `RuleX` and `RuleY` and `line` in the docs: https://github.com/search?q=repo%3Auwdata%2Fmosaic%20rulex&type=code but not sure how to enter a formula for display with the identity function.

Thank you for enabling this dashboard in the first place!! It's helping us communicate how to reduce the price of health care toward a more efficient, accessible market 🙏

**Comment by jheer** [comment]
A line mark should work well for this case. You can provide a data array with the points for your line.

For example, the data could be the literal array `[{x:0, y:0}, {x:1, y:1}]` and then you set the `x` and `y` channels accordingly. You’ll want the data values to cover the domain over which you want the line to be visualized.

Alternatively, you might look at the link mark to draw a single line segment backed by a single data point.

**Reply by jaanaltosaar** [reply]
Got it, thanks so much Jeff!! 🙏 Will give that a shot :)
